main.go

```diff
@@ -17,13 +16,14 @@ import (
 	"time"
 
 	"github.com/coreos/go-semver/semver"
+	"github.com/namsral/flag"
 )
 
 var (
 	addrFlag       = flag.String("addr", ":8080", "Serve HTTP at given address")
 	socketFlag     = flag.String("socket", "", "Serve HTTP at given UNIX socket")
-	vanityRootFlag = flag.String("vanity-root", "", "Vanity root URL (e.g.: https://upper.io).")
-	repoRootFlag   = flag.String("repo-root", "", "Git repository root URL (e.g.: https://github.com/upper).")
+	vanityRootFlag = flag.String("vanity_root", "", "Vanity root URL (e.g.: https://upper.io).")
```

Comment:
I am not sure of this change, I prefer hyphens for parameters rather than underscores. I think changing this will break scripts that currently use `-`.